modules/installation/proc-config-rhdh-custom-resource.adoc

```diff
@@ -1,4 +1,53 @@
 [id="proc-config-rhdh-custom-resource_{context}"]
 = Configuring the {product-short} Custom Resource
 
-Updates to the Backstage custom resource (CR) are automatically handled by the {product} Operator. However, updates to resources referenced by the CR, such as ConfigMaps or Secrets, are not updated automatically unless the CR itself is updated. If you want to update resources referenced by the CR, then you must manually delete the Backstage Deployment so that the Operator can re-create it with the updated resources.
\ No newline at end of file
+You can change the configuration of the Custom Resource (CR) to program the way that the Operator manages your {product} instance.
+
+[NOTE]
+====
+Updates to the Backstage CR are automatically handled by the {product} Operator. However, updates to resources referenced by the CR, such as ConfigMaps or Secrets, are not updated automatically unless the CR itself is updated. If you want to update resources referenced by the CR, then you must manually delete the Backstage Deployment so that the Operator can re-create it with the updated resources.
+====
+
+.Prerequisites
+* You have an administrative role in {ocp-brand-name}
```

Comment:
I don't think users need an admin role in OCP. A regular user with permission to create a ConfigMap and a Secret should be enough.